lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/BinarySpec.hs

```diff
@@ -325,6 +350,13 @@ instance Arbitrary TxOut where
     arbitrary = genericArbitrary
     shrink = genericShrink
 
+newtype StakeDelegationTx =
+    StakeDelegationTx (PoolId, XPub, Tx, [TxWitness])
```

Comment:
if the above suggestion is right then here `StakeCertification` should be used. We do not want to engage on this level into calculating this basing on PoolId, XPub and signing